drivers/adc/adc_shell.c

```diff
@@ -74,37 +74,72 @@ static struct adc_hdl {
 	struct adc_channel_cfg channel_config;
 	uint8_t resolution;
 } adc_list[] = {
+	/* Alphabetically sorted. */
```

Comment:
as it's interim it might not be worth the change, but @fabiobaltieri added `zephyr-keep-sorted-start` / `zephyr-keep-sorted-stop` markers to have a given block's sortedness enforced when checking compliance

Reply:
I mean... why not :-) should work...